packages/evm/src/evm.ts

```diff
@@ -129,6 +129,14 @@ export interface EVMOpts {
    * The External Interface Factory, used to build an External Interface when this is necessary
    */
   eei: EEIInterface
+
+  /**
+   * Option to disable Debug logging
```

Comment:
Update: sorry for the twists, I have yet another naming suggestion here with `deactivateCLDebug`, then set to `false` (so adopt the other comment suggestion accordingly), since this is really about deactivation of the whole debugging and not just printing.

Does that make sense?

The doc description should be a bit more descriptive, adding that the option takes precedence over the `DEBUG` CL variable usage.